packages/smart-wallet/src/walletFactory.js

```diff
@@ -246,6 +243,16 @@ export const start = async (zcf, privateArgs, baggage) => {
     },
   );
 
+  // NOTE: both `MsgWalletAction` and `MsgWalletSpendAction` arrive as BRIDGE_ID.WALLET
+  // by way of performAction() in cosmic-swingset/src/launch-chain.js
+  await (walletBridgeManager &&
+    E(walletBridgeManager).setHandler(handleWalletAction));
+
+  // for use by upgraded versions.
+  if (!baggage.has('DidStart')) {
+    baggage.init('DidStart', true);
+  }
```

Comment:
we want `DidStart` in addition to `invitationIssuer` etc.?

Reply:
Good question. It has a nice clarity but probably not worth the storage cost. Removed.